blueprints/ember-bootstrap/index.js

```diff
@@ -6,60 +6,151 @@ const fs = require('fs');
 const path = require('path');
 const writeFile = rsvp.denodeify(fs.writeFile);
 const chalk = require('chalk');
+const BuildConfigEditor = require('ember-cli-build-config-editor');
+const SilentError = require('silent-error'); // From ember-cli
```

Comment:
As I am not aware of `silent-error`, what does it do?
Also if we do use it, we should add it explicitly to our dependencies.